helm/ph-ee-engine/connector-bulk-closedloop/templates/deployment.yaml

```diff
@@ -69,7 +69,10 @@ spec:
             - name: "TENANT"
               value: "{{ .Values.global.tenant }}"
             - name: "CLOUD_AWS_REGION_STATIC"
-              value: "{{ .Values.aws.region }}"
+              valueFrom:
```

Comment:
add s3 base  URL and bucket name as well

Reply:
done